_rules/non-visual-reference-alternative-825ule.md

```diff
@@ -0,0 +1,481 @@
+---
+id: 825ule
+name: Non-visual reference alternative
+rule_type: atomic
+description: |
+  This rule checks that when there is a visual reference of content, there are also non-visual indicators of the location.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.3.3: # Sensory Characteristics (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G96: # Providing textual identification of items that otherwise rely only on sensory information to be understood.
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  -  DOM Tree
+  -  Language
+authors:
+  -  Brian Bors
+  -  Daniël Strik
+  -  Wilco Fiers
+---
+
+## Applicability
+
+Any text node that includes one of the [visual reference words](#visual-reference-words), that is [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree)
+
+## Expectation
+
+Each target that describes any [web content](https://www.w3.org/TR/WCAG21/#dfn-content) through the use of the [visual reference words](#visual-reference-words), is included in a [section of the content](#section-of-content) that also describes that [web content](https://www.w3.org/TR/WCAG21/#dfn-content) by a non-visual property, except if the target is not part of an instruction about [web content](https://www.w3.org/TR/WCAG21/#dfn-content) or the visual reference word is included in the described content.
```

Comment:
I think that "described content" needs to be better defined. Is that the visible content, the programmatically determinable content, both?

Reply:
I don't think it matters. As long as it is content (which is well defined I would say) and it is described by the target. What situation would be ambiguous?